.rake_tasks~

```diff
@@ -0,0 +1,60 @@
+about
```

Comment:
What is the file for?
`Did you check the code on github yourself before asking for review?`
Written in the following page.
https://github.com/jp-ryuji/playbook/wiki/Self-Review-Check-List